linera-views/tests/admin_tests.rs

```diff
@@ -7,27 +7,53 @@ use linera_views::dynamo_db::DynamoDbStore;
 use linera_views::rocks_db::RocksDbStore;
 #[cfg(with_scylladb)]
 use linera_views::scylla_db::ScyllaDbStore;
-use linera_views::{memory::MemoryStore, test_utils::admin_test};
+use linera_views::{
+    memory::MemoryStore,
+    test_utils::{namespace_admin_test, root_key_admin_test},
+};
 
 #[tokio::test]
-async fn admin_test_memory() {
-    admin_test::<MemoryStore>().await;
+async fn namespace_admin_test_memory() {
+    namespace_admin_test::<MemoryStore>().await;
 }
 
 #[cfg(with_rocksdb)]
 #[tokio::test]
-async fn admin_test_rocks_db() {
-    admin_test::<RocksDbStore>().await;
+async fn namespace_admin_test_rocks_db() {
+    namespace_admin_test::<RocksDbStore>().await;
 }
 
 #[cfg(with_dynamodb)]
 #[tokio::test]
-async fn admin_test_dynamo_db() {
-    admin_test::<DynamoDbStore>().await;
+async fn namespace_admin_test_dynamo_db() {
+    namespace_admin_test::<DynamoDbStore>().await;
 }
 
 #[cfg(with_scylladb)]
 #[tokio::test]
-async fn admin_test_scylla_db() {
-    admin_test::<ScyllaDbStore>().await;
+async fn namespace_admin_test_scylla_db() {
+    namespace_admin_test::<ScyllaDbStore>().await;
+}
+
+#[tokio::test]
+async fn root_key_admin_test_memory() {
+    root_key_admin_test::<MemoryStore>().await;
+}
+
+#[cfg(with_rocksdb)]
+#[tokio::test]
+async fn root_key_admin_test_rocks_db() {
+    root_key_admin_test::<RocksDbStore>().await;
+}
+
+#[cfg(with_dynamodb)]
+#[tokio::test]
+async fn root_key_admin_test_dynamo_db() {
+    root_key_admin_test::<DynamoDbStore>().await;
+}
+
+#[cfg(with_scylladb)]
```

Comment:
Might be more readable to use `test_case` for this.

Reply:
Yes, thank you.